release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -432,6 +438,14 @@ module openconfig-interfaces {
         the specified mode.";
     }
 
+    leaf hashing-policy {
+        type leafref {
+            path "/oc-hashing:policies/oc-hashing:policy/oc-hashing:name";
+        }
+        description "The hashing policy to be used when hashing packets coming
+            in on the interface.";
+    }
+
```

Comment:
I only see that Arista allows for some hash configuration on per ingress interfaces basis. And it is not clear form referenced documentation on which platform/chipset.

Reply:
The intent here is that this is a list of policies.  We'll update the description to not reference 'packets coming in the interface'.      gPINS will add support for multiple policies and Arista EOS is an existing implementation

Reply:
Can you link to the documentation that shows this being configurable per ingress /interface/ please? It's not clear to me that this is widely supported.

Reply:
Per interface is not widely supported.  gPINS plans to add support for this to Sonic.    Should we include this leaf in the model or only augment /interfaces in a gPINS/SONIC yang?  Please follow up with comments at #959